### PR TITLE
OSS-199

### PR DESCRIPTION
**Pull request by zsigmond-czine-everit**

### CODE REVIEW

File: src/main/java/org/everit/email/store/api/EmailStore.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package org.everit.email.store.api;
```
Comment (balazs-zsoldos) [inline src/main/java/org/everit/email/store/api/EmailStore.java]: no api in the package, just org.everit.email.store

File: src/main/java/org/everit/email/store/api/EmailStore.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+import org.everit.email.Email;
+
+/**
+ * A store that can persist email.
```
Comment (balazs-zsoldos) [inline src/main/java/org/everit/email/store/api/EmailStore.java]: Persistent store of emails.

File: src/main/java/org/everit/email/store/api/EmailStore.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ */
+public interface EmailStore {
+
+  /**
+   * Deletes an email permanently.
+   *
+   * @param emailStoreId
+   *          The identifier of the stored email that should be deleted.
+   */
+  void delete(long emailStoreId);
```
Comment (balazs-zsoldos) [inline src/main/java/org/everit/email/store/api/EmailStore.java]: param should be emailId, not emailStoreId.

File: src/main/java/org/everit/email/store/api/EmailStore.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+  /**
+   * Reads email based on email store identifier.
+   *
+   * @param emailStoreId
+   *          the identifier of stored email.
+   * @return the stored email. Return <code>null</code> if not found email.
+   */
+  Email read(long emailStoreId);
```
Comment (balazs-zsoldos) [inline src/main/java/org/everit/email/store/api/EmailStore.java]: Param should be emailId not emailStoreId.

**Comment by zsigmond-czine-everit** [comment]
@balazs-zsoldos: code updated.

**Pull request merged**